operator/v1/types_ingress.go

```diff
@@ -633,8 +635,15 @@ type AWSClassicLoadBalancerParameters struct {
 }
 
 // AWSNetworkLoadBalancerParameters holds configuration parameters for an
-// AWS Network load balancer.
+// AWS Network load balancer. For Example: Setting AWS EIPs https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html
```

Comment:
This "for example" and including a link seems quite odd to me for this type of GO doc. We don't list examples in `AWSClassicLoadBalancerParameters`. It seems a bit inconsistent.

Reply:
OK but I thought we should add some details like what can be added in this type.